### ISSUE TITLE
Scoop updates/cleanups don't seem to work with certain app versions

### ISSUE THREAD

**Issue by watkinsmatthewp** [lukesampson/scoop#3812]
Here are the steps to reproduce:

I'm running latest version of scoop:

```
Current Scoop version:
a9fa775d (HEAD -> master, origin/master, origin/HEAD) chore(buckets.json): Update scoop-nonportable URL (#3776)
```

When I run `scoop status` shows I have IntelliJ-IDEA v 2019.3-193.5233.102 installed.

But when I open IntelliJ it tells me there's an update.

So I run `scoop update *` and scoop reports:

```
IntelliJ-IDEA: 2019.3-193.5233.102 (latest version)
Latest versions for all apps are installed! For more information try 'scoop status'
```

So, based on the workaround mentioned in #3644, I run `scoop update -f IntelliJ-IDEA`. Scoop dutifully updates IntelliJ to version 2019.3.1-193.5662.53.

But I don't want the old version on my machine, so I run a cleanup `scoop cleanup *`, only to watch scoop uninstall the new version that it just installed!

```
Removing IntelliJ-IDEA: 2019.3.1-193.5662.53
Everything is shiny now!
```

No, scoop, everything is _not_ shiny now.

So my question is, why is scoop not registering the new version as new? I thought that was supposed to be fixed with #3644? And until that is fixed should I just never run cleanup?

Thanks!

**Comment by Ash258** [comment]
Duplicate/Related #2720

Being fixed in #3721

**Comment by rasa** [comment]
Obsolete.
